### DISCUSSION TITLE
Is there any module supporting the customized font which is installed in the device just like page.insert_font in PyPDF2?

### DISCUSSION THREAD

**Discussion by hachinoone** [py-pdf/pypdf#2439] [Q&A]
I would like to use a customized font in the annotation.

Could you please provide a module supporting such a requirement? Many thanks!

**Comment by stefan6419846** [comment]
Where did you find the documentation for `page.insert_font` in PyPDF2?

**Comment by Vini09090** [comment]
Is there a way to take a file using this , and cut it into several parts, forming new files?
